azure/worker/dispatcher.py

```diff
@@ -319,7 +321,12 @@ def __run_sync_func(self, invocation_id, func, params):
             _invocation_id_local.v = None
 
     def __poll_grpc(self):
-        channel = grpc.insecure_channel(f'{self._host}:{self._port}')
+        options = []
+        if self._grpc_max_msg_len:
+            options.append(('grpc.max_message_length', self._grpc_max_msg_len))
```

Comment:
On c#, setting max message length is deprecated. We had to set max receive length and max send length. Same might be true in python. You need to set grpc.max_send_message_length and grpc.max_receive_message_length

Reply:
Fixed.